sponsorship_compassion/views/sponsorship_contract_view.xml

```diff
@@ -79,6 +79,44 @@
         </field>
     </record>
 
+    <record id="view_compassion_sponsorship_pivot" model="ir.ui.view">
+        <field name="name">compassion.sponsorship.line.pivot</field>
+        <field name="model">recurring.contract</field>
+        <field name="arch" type="xml">
+            <pivot>
+                <field name="start_date" interval="week" type="col"/>
+                <field name="state" type="row"/>
+            </pivot>
+        </field>
+    </record>
+
+    <record id="view_compassion_sponsorship_graph" model="ir.ui.view">
+        <field name="name">compassion.sponsorship.line.graph</field>
+        <field name="model">recurring.contract</field>
+        <field name="arch" type="xml">
+            <graph>
+                <field name="start_date" interval="week" type="col"/>
+                <field name="state" type="row"/>
+            </graph>
+        </field>
+    </record>
+
+    <record id="view_recurring_contract_filter_graph" model="ir.ui.view">
+        <field name="name">recurring.contract.graph.select</field>
+        <field name="model">recurring.contract</field>
+        <field name="inherit_id" ref="contract_compassion.view_recurring_contract_compassion_filter" />
+        <field name="arch" type="xml">
+            <filter name="english" position="after">
+               <filter string="Last Fiscal Year" name="last_fiscal_year"
+                    domain="[('start_date', '&gt;=' ,(context_today()-relativedelta(years=1)).strftime('%Y-07-01')),

```

Comment:
If we are before July, this filter will return the current Fiscal Year instead of Last Fiscal Year